app/controllers/orders_controller.rb

```diff
@@ -25,13 +24,20 @@ def create
       end
       flash[:success] = "Your order has been created"
       session.delete(:cart)
-      redirect_to "/orders/#{order.id}"
+      redirect_to "/profile/orders/#{order.id}"
```

Comment:
I did this all over the place, so now the default order show page is for a regular user